app/scss/adminPage.scss

```diff
@@ -83,6 +83,14 @@ input {
 .table {
   font-size: 1.5rem;
 
+  .table-buttons {
+    width: 25%;
+  }
+
+  .table-columns {
```

Comment:
check your maths. there are 9 `table-columns` and 1 `table-buttons` and they need to add up to 100%, currently they add up to 115%

Reply:
we decided to go with the 'auto' because using the scss percentage function made it look UGLY :)